src/vcf2fasta.py

```diff
@@ -249,7 +329,14 @@ def translate_genome(genome, encoding="nd16", phased=True):
 
 if __name__ == "__main__":
     args = parse_args()
+    debug_mode = args.debug
```

Comment:
Fix: Move anything that is not argument parsing into the `vcf2fasta` function.

---

User-interfaces are notoriously hard to debug. CLI is still fine, but you see that to do that, you need to run Python in a subprocess and catch its output.

A better way to do this is to isolate any functionality into a module (`vcf2fasta()`), and have CLI as a thin wrapper. This thin wrapper can be user-tested or whatever, but only thin it does is parsing command-line arguments and sending them to the isolated module.

Since the isolated module now encompasses _all_ other functionality, it can be tested without doing any shenanigans because, well, its just a function (or class) without other dependencies other than some input arguments.